tdrs-frontend/src/actions/requestAccess.js

```diff
@@ -14,9 +15,7 @@ export const requestAccess =
     try {
       const URL = `${process.env.REACT_APP_BACKEND_URL}/users/request_access/`
       const user = { first_name: firstName, last_name: lastName, stt: stt?.id }
-      const { data } = await (
-        await axiosInstance
-      ).patch(URL, user, {
+      const { data } = await axios.patch(URL, user, {
```

Comment:
just wanted to point out why this change was needed. `axiosInstance`, for whatever reason, didn't preserve the default `axios` behavior of injecting the `csrftoken` cookie into the `X-CSRFTOKEN` header. swapping it out here fixes the issue. I don't know where else `axiosInstance` is used but it will likely face the same issue with our cypress tests. Luckily, there doesn't seem to be any configuration on it, so swapping it out for the default should be safe (and seemingly adds functionality 😄 )